package/MDAnalysis/core/groups.py

```diff
@@ -975,6 +980,15 @@ class AtomGroup(GroupBase):
     .. SeeAlso:: :class:`MDAnalysis.core.universe.Universe`
 
     """
+    def __getitem__(self, item):
+        # u.atoms['HT1'] access, otherwise default
+        if isinstance(item, string_types):
```

Comment:
https://github.com/MDAnalysis/mdanalysis/blob/master/package/MDAnalysis/core/AtomGroup.py#L3524

It was in the previous incarnation of docs, so it's part of the tech debt in #1078